mixes/helpers/get-tracks.js

```diff
@@ -0,0 +1,21 @@
+const { filter } = require('lodash')
+const { CHANNEL_TYPE_PRIMARY_TRACK } = require('../../channels/constants')
+
+module.exports = {
+  getPrimaryTracks,
+}
+
+function getPrimaryTracks (mix, metas) {
+  const allChannels = (mix && mix.channel && mix.channel.channels) || []
```

Comment:
similar questions around defensiveness. seems like boilerplate but also a nice to have because the functions are totally pure? so they dont care about data full load or partial load or anything?